### ISSUE TITLE
Please help with validation errors on inline or block buttons

### ISSUE THREAD

**Issue by benji89** [CalderaWP/Caldera-Forms#2081]
## Do You Need Immediate Help?
NO

## Is This A Feature Request?
NO

## What Version Of Caldera Forms, WordPress and PHP Are You Using?
WordPress Version: 4.7.7PHP Version: 5.3.8MySQL Version: 5.1.56Caldera Forms Version: 1.5.6.1WP_DEBUG:

## Does Your Issue Persist When You Disable All Other Plugins and Switch To The Default Theme?
Yes

## What Is The Unexpected Behaviour?
* Please explain what you did, what you expected to happen, and what happened instead.
Hello there, I have an issue with validation errors: I set mandatory inline radio box. 

When I don't answer a mandatory inlined radio box question, I have an error message "This value is required." that is placed in between the radio buttons (see pic below). Same apply for block buttons.

![capture d ecran 2017-11-15 a 15 00 28](https://user-images.githubusercontent.com/28863413/32839757-c2cb70f6-ca15-11e7-8b54-b1bcca462d8e.png)

You can try here: http://www.bouhanna.com/diagnostic-greffe/

## What PHP Errors Have You Logged While Reproducing This Bug?
Didn't succeed to config it

## What JavaScript Errors Have You Seen While Reproducing This Bug?
![capture d ecran 2017-11-15 a 15 05 22](https://user-images.githubusercontent.com/28863413/32839949-70f99de2-ca16-11e7-8b97-75cf9070d238.png)

Thank you in advance for your help!

**Comment by Shelob9** [comment]
@benji89  Thanks for using Caldera Forms. Can you please let us know if the layout issues continue with the default theme or if this is an issue with your theme?

Also, do those errors happen if you remove auto-optimize, if so, what are the source files that cause those errors?

**Comment by ictbeheer** [comment]
@Shelob9 this seems like the same issue I reported in #1700. I was still able to reproduce the error with only Caldera Forms actived and the default theme.

**Comment by Shelob9** [comment]
@ictbeheer Yes, that's the same issue. I will work on this once #2091 is merged.
